libevmcore/Assembly.h

```diff
@@ -24,6 +24,7 @@
 #include <iostream>
 #include <sstream>
 #include <libdevcore/Common.h>
+#include <libdevcore/SourceLocation.h>
```

Comment:
I think devcore is fine. It's not EVM related. A source location can be used anywhere, from Solidity and evm assemgbly items to the Mix editor.